media/linux/pds-sqlite3-queries/create-training-rosters.py

```diff
@@ -52,13 +63,36 @@ def pretty_member(member):
             val = '{ph} {type}'.format(ph=phone['number'], type=phone['type'])
             phones.append(val)
 
+    ministries = {'weekday'     :   'No',
+                  'weekend'     :   'No',
+                  'homebound'   :   'No',}
+    key = 'active_ministries'
+    if key in member:
+        for ministry in member[key]:
+            if ministry['Description'] == 'Weekend Communion':
+                ministries['weekend'] = 'Yes'
+            if ministry['Description'] == 'Weekday Communion':
+                ministries['weekday'] = 'Yes'
+            if ministry['Description'] == 'Homebound Communion':
+                ministries['homebound'] = 'Yes'
+
     email = PDSChurch.find_any_email(member)[0]
 
+    if (int)(member['family']['ParKey']) > 9000:
+        active = "No"
+    else:
+        active = "Yes"
+
     m = {
         'mid'       :   member['MemRecNum'],
         'name'      :   member['first']+' '+member['last'],
+        'email'     :   email,
         'phones'    :   phones,
-        'email'     :   email,}
+        'active'    :   active,
+        'weekend'   :   ministries['weekend'],
+        'weekday'   :   ministries['weekday'],
+        'homebound' :   ministries['homebound']
+    }
     
     return m
 
```

Comment:
What is `_dt_to_int()` for?  I don't see it used anywhere in the code.

Also, if you need a numeric representation of the date, you should use `datetime.timestamp()`, which returns an integer number of seconds since the epoch (Jan 1 1970).